basic.css

```diff
@@ -0,0 +1,86 @@
+html {
+  height: 100%;
+}
```

Comment:
- [ ] Kindly remove the default margin and padding of the entire website as a default style.
- Add these lines to your html style
```mdx
html{
  .............
  margin: 0;
  padding: 0;
}
```
The reason is that your page is having white border all around the app. Please fix that. 
![image](https://user-images.githubusercontent.com/50941074/164761246-12ccafcd-7e57-4649-bfb2-c6a8d98497f8.png)